platform/android/java_class_wrapper.cpp

```diff
@@ -410,7 +617,21 @@ bool JavaClass::_call_method(JavaObject *p_instance, const StringName &p_method,
 				to_free.push_back(jarr);
 			} break;
 			case ARG_ARRAY_BIT | ARG_TYPE_CLASS: {
-				argv[i].l = nullptr;
+				String cn = method->param_sigs[i].operator String();
+				if (cn.begins_with("[L") && cn.ends_with(";")) {
+					cn = cn.substr(2, cn.length() - 3);
+				}
+				jclass c = env->FindClass(cn.utf8().get_data());
```

Comment:
We should check that `c` is valid before proceeding.

Reply:
We would have checked this earlier on line 224.

This process goes in two phase: a validation phase, where it reports errors about any of the provided arguments, and then a calling phase where (if there were no validation errors) it marshals the data into the Java types and actually calls the method. This would have gotten caught in the validation phase.

However, it's no problem to add `if (c) { ... }`, so I have done that in my latest push :-)